pkg/network/server_test.go

```diff
@@ -389,7 +389,7 @@ func startTestServer(t *testing.T, protocolCfg ...func(*config.Blockchain)) *Ser
 }
 
 func startWithCleanup(t *testing.T, s *Server) {
-	go s.Start()
+	s.Start()
```

Comment:
Describe what has been changed in the server's `Start` behaviour in the commit message.